public/static/locales/en/common.json

```diff
@@ -229,5 +229,55 @@
   "minimum_booking_notice": "Minimum booking notice",
   "delete_event_type_description": "Are you sure you want to delete this event type? Anyone who you&apos;ve shared this link with will no longer be able to book using it.",
   "delete_event_type": "Delete Event Type",
-  "confirm_delete_event_type": "Yes, delete event type"
+  "confirm_delete_event_type": "Yes, delete event type",
+  "connect_new_app": "Connect a new App",
+  "integrate_your_account_services": "Integrate your account with other services.",
+  "add": "Add",
+  "select_calendars": "Select Calendars",
+  "if_no_entry_all_calendars_checked": "If no entry is selected, all calendars will be checked",
+  "connect_caldav_server": "Connect to CalDav Server",
+  "your_credentials_stored_encrypted": "Your credentials will be stored and encrypted.",
+  "connect_apple_server": "Connect to Apple Server",
+  "generate_app_spec_password": "Generate an app specific password to use with cal.com at",
+  "integrations": "Integrations",
+  "connect_favourite_apps": "Connect your favourite apps.",
+  "calendar_integration": "Calendar Integration",
+  "video_conferencing": "Video Conferencing",
+  "connected": "Connected",
+  "not_connected": "Not Connected",
+  "dont_have_apps_connected": "You don&apos;t have any apps connected.",
+  "dont_have_apps_connected_description": "You currently do not have any apps connected. Connect your first app to get started.",
+  "select_which_calendars_checked_double_booking": "Select which calendars are checked for availability to prevent double bookings.",
+  "launch_own_app": "Launch your own App",
+  "if_want_add_your_app_touch_us": "If you want to add your own App here, get in touch with us.",
+  "contact_us": "Contact us",
+  "integrations_details": "Integrations Details",
+  "manage_delete_this_app": "Manage and delete this app.",
+  "name_app_prop": "{`${getIntegrationName(props.integration.type)} App`}",
```

Comment:
This will never work, you must use dynamic variables (check below comment).

Reply:
```json
  "name_app_prop": "{{integrationName}} App",
```
then
```typescript
{t('name_app_prop', { integrationName: getIntegrationName(props.integration.type) })
```
inside the component.

For other languages it can be changed based on the language (this will be done in crowdin).
For example:
```json
"name_app_prop": "{{integrationName}} Aplicación",
```
or
```json
"name_app_prop": "{{integrationName}} Aplicação",
```
even the order can be different:
```json
"name_app_prop": "Aplicação {{integrationName}}",
```